### DISCUSSION TITLE
Have you ever experienced issues with emojis not matching across systems/machines?

### DISCUSSION THREAD

**Discussion by carlotrimarchi** [BryanJenksCommunity/FAQ#113] [Q&A] [answered]
I'm just starting with Obsidian and today I was having a look at your templates that I received after subscribing to the newsletter.

I noticed that the emojis in your templates (for example `📝️/🌱️`), don't match the ones that I insert manually through my Mac (`📝/🌱`). You should be able to verify this yourself by comparing the 2 sequences in a the JavaScript console

```
`📝️/🌱️` === `📝/🌱`; // this should return false (the first sequence comes from one of your templates, the second from me
`📝️/🌱️` === `📝️/🌱️`; // this should return true (they are the same sequence)
```

Any idea why?

**Comment by tallguyjenks** [comment, marked as answer]
yeah, there's some wonky-ness some times when you hit a back space at the end of one of the emoji's but it wont actually remove the glyph itself but it just doesn't match anymore. It's weird. One quick fix would be to just use a `sed` mass bulk replace on any of those outliers. that or VS Code
